apps/builder/app/builder/features/sidebar-left/panels/components/scroll-area.tsx

```diff
@@ -0,0 +1,78 @@
+import { styled, theme } from "@webstudio-is/design-system";
+import {
+  Root,
+  Viewport,
+  Scrollbar,
+  Thumb,
+  Corner,
+} from "@radix-ui/react-scroll-area";
+
+const SCROLLBAR_SIZE = 10;
+
+const ScrollAreaRoot = styled(Root, {
+  borderRadius: 4,
+  overflow: "hidden",
+  boxShadow: `0 2px 10p ${theme.colors.blackA7}`,
+  backgroundColor: "white",
+});
+
+const ScrollAreaViewport = styled(Viewport, {
+  width: "100%",
+  height: "100%",
+  borderRadius: "inherit",
+});
+
+const ScrollAreaScrollbar = styled(Scrollbar, {
+  display: "flex",
+  // ensures no selection
+  userSelect: "none",
+  // disable browser handling of all panning and zooming gestures on touch devices
+  touchAction: "none",
+  padding: 2,
+  background: theme.colors.blackA6,
+  transition: "background 160ms ease-out",
+  "&:hover": { background: theme.colors.blackA8 },
+  '&[data-orientation="vertical"]': { width: SCROLLBAR_SIZE },
+  '&[data-orientation="horizontal"]': {
+    flexDirection: "column",
+    height: SCROLLBAR_SIZE,
+  },
+});
```

Comment:
Doesn't radix provide all necessary style?

Reply:
no, I will move it to design-system, radix never provides styles

Reply:
these styles are copied from example, I will update them to use native macos styles

Reply:
or Taylor will design something